docs/arcaflow/workflows/schemas.md

```diff
@@ -0,0 +1,61 @@
+# Workflow Schemas
+
+## Schema Names
+
+### Non-Object Names
+
+Schemas that are not composed within an ObjectSchema do not have an `Object ID`. They use a stringified version of their `TypeID` for their schema name.
+
+| schema        |     type id                 |
+|---------------|-----------------------------|
+| IntEnum       | enum_string                      |
+| StringEnum    | enum_integer                      |
+| String        | string                      |
+| Pattern       | pattern                      |
+| Int           | integer                     |
+| Float         | float                    |
+| Bool          | bool                      |
+| Map           | map              |
+| OneOfString   | one_of_string   |
+| OneOfInt      | one_of_int      |
+
+`ListSchema`s use the `ListSchema` `TypeID` with the schema name of their constituent element, or item value. `ListSchema` names are prepended with `list` until they reach a non-`ListSchema` item type. These names are joined with a single underscore, `_`.
+
+| schema              |  name                 |

```

Comment:
Should this be "schema name" and "schema ID"?

(I'm really confused as to what is a "schema name" and what is a "schema ID"....)